CactbotOverlay/CactbotOverlay.csproj

```diff
@@ -51,26 +51,25 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Advanced Combat Tracker">
-      <HintPath>ThirdParty\ACT\Advanced Combat Tracker.exe</HintPath>
+      <HintPath>..\..\OverlayPlugin-Compiler\OverlayPlugin\OverlayPlugin\bin\x64\Release\Advanced Combat Tracker.exe</HintPath>
```

Comment:
These whole file seems like accidental changes.

Reply:
yes, this is accidental, was rushing and forgot to exclude this from the commit.

Reply:
I've fixed this, my next pull *should* be missing this file, or at least have no changes.